src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -168,6 +169,15 @@ Learn more by reading [Establish network connections early to improve perceived
 speed](/preconnect-and-dns-prefetch/)
 {% endAside %}
 
+### Use signed exchanges (SXGs)
+
+[Signed exchanges (SXGs)](https://web.dev/signed-exchanges) are a delivery mechanism that allow for
+faster user experiences by providing content in an easily cacheable format.
+Specifically, [Google Search](https://developers.google.com/search/docs/advanced/experience/signed-exchange)
+will cache and sometimes prefetch SXGs. For sites that receive a large portion
+of their traffic from Google Search, SXGs can be an important tool for improving
+LCP. For more information, see [Signed
+Exchanges](https://web.dev/signed-exchanges).
```

Comment:
```suggestion
Exchanges](/signed-exchanges).

```

(Re: the extra space) Some Markdown parsers need a space here to detect the new section heading